tool/tsh/tsh.go

```diff
@@ -608,8 +611,10 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		EnumVar(&cf.MFAMode, modes...)
 	app.HelpFlag.Short('h')
 
-	ver := app.Command("version", "Print the version of your tsh binary")
+	ver := app.Command("version", "Print the version of your tsh binary and version of the Teleport proxy in current tsh profile")
 	ver.Flag("format", defaults.FormatFlagDescription(defaults.DefaultFormats...)).Short('f').Default(teleport.Text).EnumVar(&cf.Format, defaults.DefaultFormats...)
+	ver.Flag("client", "Show the client version only (no server required).").
```

Comment:
Please use the same terminology in the CLI reference doc too.

Reply:
UPdated